### DISCUSSION TITLE
Titles on Webpages

### DISCUSSION THREAD

**Discussion by wgmayer0** [qutebrowser/qutebrowser#7113] [Q&A] [answered]
I am using an [xmonad module](https://hackage.haskell.org/package/xmonad-contrib-0.13/docs/XMonad-Actions-GridSelect.html) that tells me the titles of windows that are currently open.

I see the names of other things I have open, but the qutebrowser windows are all labeled: `/`. I found a setting in `autoconfig.yml` that says:

```
  window.title_format:
    global: /
```

How could I get the title format to be the title of the webpage? I guess I would like it to match whatever the `<title></title>` value is for the webpage that window is on.

**Comment by wgmayer0** [comment, marked as answer]
I found the solution:

```
  window.title_format:
    global: '{current_title}'

```

**Reply by mschilli87** [reply]
Great you found it. Note that the default is `'{perc}{current_title}{title_sep}qutebrowser'` though. So maybe you (accidentally) changed it via `qute://settings` or by running `:set window.title_format /`? To restore the default, run `:config-unset window.title_format`.

**Comment by wgmayer0** [comment]
Thank you!
